lib/common/client/concerns/mhv_session_based_client.rb

```diff
@@ -33,15 +33,15 @@ def client_session(klass = nil)
         end
       end
 
+      def token_headers
```

Comment:
I'm not crazy about this being made a public method. Would almost prefer if we use .send to invoke it than to make it a public method. If StreamingClient was a monkey patch of Faraday or a mixin for Common::Client rather than its own class, this might not be necessary at all.